makefile

```diff
@@ -15,4 +15,7 @@ console:
 	env $$(cat .env) irb -r ./app
 
 db:
-	env $$(cat .env) ruby create_table.rb
+	env $$(cat .env) ruby ./db/create_table.rb
+
+test:
```

Comment:
If I try to run this, I get the following error:

```
Erics-MacBook-Pro:actionite ericboersma$ make test
makefile:21: *** missing separator.  Stop.
```
@ceciliarivero Any thoughts?